flash/core/utilities/imports.py

```diff
@@ -99,5 +98,31 @@ def _compare_version(package: str, op, version) -> bool:
     _MATPLOTLIB_AVAILABLE,
     _COCO_AVAILABLE,
     _FIFTYONE_AVAILABLE,
+    _PYSTICHE_AVAILABLE,
 ])
 _SERVE_AVAILABLE = _FASTAPI_AVAILABLE and _PYDANTIC_AVAILABLE and _CYTOOLZ_AVAILABLE and _UVICORN_AVAILABLE
+
+_EXTRAS_AVAILABLE = {
+    'image': _IMAGE_AVAILABLE,
+    'tabular': _TABULAR_AVAILABLE,
+    'text': _TEXT_AVAILABLE,
+    'video': _VIDEO_AVAILABLE,
+    'serve': _SERVE_AVAILABLE,
+}
+
+
+def _requires_extras(extras: str):
```

Comment:
who about multiple? double wrapper or just allow `Union[str, Sequence[str]]`

Reply:
Currently we just support requiring a single "extras" install. It would only make sens to add support for multiple when / if we have a use case for it.

Reply:
basically I aim at the granularity that the user may have installed some packages without Flash defined extras so for example for this function he does need all extras for image, right?